src/xrSound/stdafx.cpp

```diff
@@ -15,4 +15,6 @@
 #pragma comment(lib, "OpenAL32B.lib")
 #pragma comment(lib, "dsoundb.lib")
 //#pragma comment(lib, "xrAPI.lib")
+#elif DEBUG
+#pragma comment(lib, "xrCDB.lib") // Xottab_DUTY: For some reason it wont compile without it.
```

Comment:
Without direct point to xrCDB.lib xrSound wont compile. I can put this in project file, but I think we should investigate in future into this (Why in debug just only project reference wont work, just like in release). So, it's a reminder in the future.